modules/standard/HDFS.chpl

```diff
@@ -18,19 +18,315 @@
  */
 
 /*
-   HDFS module
+This module implements support for the 
+`Hadoop <http://hadoop.apache.org/>`_ 
+`Distributed Filesystem <http://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-hdfs/HdfsUserGuide.html>`_ (HDFS).
+
+Dependencies
+------------
+
+The HDFS functionality in Chapel is dependent upon both Hadoop and Java being
+installed.  Your ``HADOOP_INSTALL``, ``JAVA_INSTALL`` and ``CLASSPATH``
+environment variables must be set as described below in
+:ref:`setting-up-hadoop`.  Without this it will not compile with HDFS, even if
+the flags are set. As well, the HDFS functionality is also dependent upon the
+``CHPL_AUXIO_INCLUDE`` and ``CHPL_AUXIO_LIBS`` being set properly. For more
```

Comment:
There is either an extra "the" or a missing phrase "environment variables" here.